extensions/cli/debug/src/main/java/mil/nga/giat/geowave/cli/debug/CQLDelete.java

```diff
@@ -0,0 +1,92 @@
+package mil.nga.giat.geowave.cli.debug;
```

Comment:
is this duplicative of vector CQLDelete?  Is there a reason to have this separately?